uint8 type
uint32 count
---
uint8 result
